etherhoused-log

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exec papertrail --min-time "24 hours ago" "program:app/web.1 AND (\"turn_off ${1}\" OR \"turn_on ${1}\")"
```

Comment:
I guess. Papertrail is meant to be run by humans.
I'm not aware of a function for a heroku app to get it's own logs.

A syslog version is fine. It wouldn't work in production, which is fine. 
My only objection is having papertrail run by the server. This is never a working situation.
Personally having logs exposed like this as an endpoint in production sounds crazy to me, but I'm fine for it for debug.

But yea having the a etherhoused-log binary that just says "echo 'put your log fetching command here'" or gitignoring it is fine.